web/react/stores/user_store.jsx

```diff
@@ -3,7 +3,6 @@
 
 var AppDispatcher = require('../dispatcher/app_dispatcher.jsx');
 var EventEmitter = require('events').EventEmitter;
-var assign = require('object-assign');
```

Comment:
We might want to check out if we can remove object-assign completely from our project since I don't think I've seen it in other places. Babelify may also be able to handle the actual ES6 Object.assign so even if we have a use for it, we might not need the polyfill.